csrc/scheduler/vectorize_helper.h

```diff
@@ -158,16 +158,19 @@ class TORCH_CUDA_CU_API ProjectedExtent {
   // is_divisible_. This is important because we might use expr evaluator to
   // evaluate their values multiple times, so we want to simplify them once and
   // cache the result so that later evaluation can save time.
-  void finalize() {
+  void finalize(const std::vector<Bool*>& assumptions) {
     if (quotient_ == nullptr) {
       // TODO: print the simplified expression and see if expr simplifier needs
       // improvement
-      quotient_ = simplifyExpr(quotient());
+      quotient_ = simplifyExpr(quotient(), {}, assumptions);
+      std::cout << "quotient: " << quotient_->toInlineString() << std::endl;
```

Comment:
Keeping this print for now for convenient of debugging https://github.com/NVIDIA/Fuser/issues/408, will remove before merge.